docs/UserGuide.md

```diff
@@ -127,72 +122,27 @@ Expected behaviour upon failure:
 - Phone number format invalid: Displays error message “Please specify the guest’s phone number with only numbers with no spaces or special characters”.
 - `r/` tag uses an invalid value: Displays error message “RSVP status can only be `yes`, `no` or `unknown`”.
 
+<span style="font-size: 0.4em;">[Back to Top](#wedlog-user-guide)</span>
 --------------------------------------------------------------------------------------------------------------------
-### Deleting a guest: `guest delete`
-Deletes the specified guest from WedLog.
-
-```text
-guest delete INDEX
-```
-
-Acceptable values for `INDEX`:
-- A positive integer
-
-Examples:
-- `guest delete 2` deletes the 2nd guest on the guest list
-- `guest find Betsy` followed by `guest delete 1` deletes the 1st guest in the results of the `find` command
-
-Expected behaviour upon success:
-- Deletes the person at the specified `INDEX`
-  - If the previous command was `guest filter KEY_WORDS`, the `INDEX` refers to the index number shown in the filtered guest list
-  - Otherwise, the `INDEX` refers to the index number on the unfiltered guest list
-- Displays a message telling user which guest has been deleted
 
-Expected behaviour upon failure:
-- Number out of index range, not a number, or no number: Displays error message “Please input a positive integer as the index.”
-- Number does not correspond to any guest: Displays error message “The index you have provided does not correspond to any guest.”
-- No input index: Displays error message “Please input an index.”
-
---------------------------------------------------------------------------------------------------------------------
-### Adding a vendor : `vendor add`
+### Adding a vendor: `vendor add`
 
 Adds a vendor to WedLog.
 
 ```text
-vendor add n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG...]
+vendor add n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG][...​](#ZWSP-anchor)
 ```
-A guest must have the following tags: `n/NAME`
-
-The following tags are optional: `p/PHONE_NUMBER e/EMAIL a/ADDRESS r/RSVP_STATUS d/DIETARY_REQUIREMENTS t/TAG...`
-
-Acceptable values for `n/NAME`:
-- Alphanumeric word with or without spaces
-
-Acceptable values for `n/PHONE_NUMBER`:
-- Numbers with no spaces or special characters
-
-Acceptable values for `e/EMAIL`:
-- `local-part@domain`
-  - the `local-part` must:
-    - contain alphanumeric characters and these special characters, excluding the parentheses (+_.-)
-    - not start or end with any special characters
-  - the `domain` must:
-    - consist of domain labels separated by periods
-    - end with a domain label at least 2 characters long
-    - have each domain label start and end with alphanumeric characters
-    - have each domain label consist of alphanumeric characters, separated only by hyphens, if any
+A guest must have the following parameter: `n/NAME`
```

Comment:
do you think this line is pretty common sense? it's already indicated with the square brackets above too